mutt/regex.c

```diff
@@ -216,52 +218,42 @@ bool mutt_regexlist_match(struct RegexList *rl, const char *str)
  * mutt_regexlist_new - Create a new RegexList
  * @retval ptr New RegexList object
  */
-struct RegexList *mutt_regexlist_new(void)
+struct RegexListNode *mutt_regexlist_new(void)
 {
-  return mutt_mem_calloc(1, sizeof(struct RegexList));
+  return mutt_mem_calloc(1, sizeof(struct RegexListNode));
 }
 
 /**
  * mutt_regexlist_remove - Remove a Regex from a list
- * @param rl  RegexList to alter
+ * @param head  RegexList to alter
```

Comment:
I prefer the original `rl` here -- it's reminds the reader of the type.
The code is full of variables called `tmp` -- a temporary what?!